scripts/check_frontend_coverage_test.py

```diff
@@ -204,14 +201,11 @@ def test_check_coverage_changes_insertion(self):
             with fully_covered_tests_swap:
                 with self.assertRaises(SystemExit) as error:          
                     check_frontend_coverage.check_coverage_changes()
-                self.assertEqual(error.exception.code, 1)
-                self.assertEqual(
```

Comment:
I notice you're deleting a bunch of the assertions to mock_print. However, there are two separate things happening: the function is raising an error to the parent process, and the function is printing logs to the terminal. You should have tests for both of these (both the content of the exception, as well as what is shown to the user). If no logs are printed (because the exception already has the error information), assert that too.

Reply:
I've removed all the `python_utils.PRINT()` from the script, in order to put the log text in `sys.exit()` and `Exception()`. The only place using the method id here:

```
if errors:
        python_utils.PRINT('------------------------------------')
        python_utils.PRINT('Frontend Coverage Checks Not Passed.')
        python_utils.PRINT('------------------------------------')
        sys.exit(errors)
    else:
        python_utils.PRINT('------------------------------------')
        python_utils.PRINT('All Frontend Coverage Checks Passed.')
        python_utils.PRINT('------------------------------------')
```
The `else` statement is being tested checking if `sys.exit()` was called. If it was not, the test pass. And either `if` and `else` statements print thrice times, so I can't difference each one was been called counting the times `python_utils.PRINT()` was called. That's why I've deleted assertions to mock_print, because I don't need it anymore to test the script.

Reply:
OK, got it. Thanks!